### PR TITLE
docs: add pull request template

### LINKED ISSUE #1596: docs: add PR template
When you add a pull request template to your repository, project contributors will automatically see the template's contents in the pull request body.

For more information read this documentation -> https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

### PR DESCRIPTION
**Pull request by YashPimple**
fixes #1596 

Here comes the changes proposed through this PR:
- [X] Created pull_request.md file as per instructed 

Signed-off-by: Yash Pimple <yashpimple22@gmail.com>

### CODE REVIEW

**Review by DavidPHirsch — APPROVED** [review]

**Review by mowies — CHANGES_REQUESTED** [review]
please put the template into the `PULL_REQUEST_TEMPLATE` folder next to the issue template folder

**Comment by YashPimple** [comment]
@mowies I have made the required changes. Please review and let me know if any further adjustments are needed

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:5]:
```suggestion
Fixes #XXXX <!-- replace with the issue number that is fixed in this PR -->
```

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:7]:
```suggestion

```

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:18]: I don't think we need checkboxes for this necessarily
```suggestion
- change one
- change two
- ...
```

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:20]:
```suggestion
## Check List (Check all the boxes that are applicable)
```

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:24]: This repo is the documentation. I don't think we need this here.

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:27]:
```suggestion
- [ ] The title of my pull request is a short description of the requested changes.
- [ ] The PR title follows semantic commit conventions.
```

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:32]:
```suggestion

```

**Review by mowies — CHANGES_REQUESTED** [review]
I added some things to change. I think we can make things a bit simpler here

**Comment by YashPimple** [comment]
@mowies I have made the required changes. Please do have a look

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:18]: i didn't mean that you need to remove the whole list here. just the lines i mentioned :)

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:7]: pls remove this line

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md
Comment (YashPimple) [reply .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:18]: Got it, I'll remove the lines you mentioned and leave the rest of the list as is. Thanks for the heads up!

File: .github/workflows/ISSUE_TEMPLATE/pull_request_template.md
Comment (mowies) [inline .github/workflows/ISSUE_TEMPLATE/pull_request_template.md:11]:
```suggestion
## Proposed changes
```

**Review by mowies — APPROVED** [review]

**Comment by mowies** [comment]
@YashPimple LGTM!
Before merging you need to add sign-off to all your commits in this PR. check the DCO check [here](https://github.com/keptn/keptn.github.io/pull/1602/checks) on how to do it if you don't know.

**Review by mowies — CHANGES_REQUESTED** [review]
Will leave this on "request changes" until you have signed all the commits :)

**Comment by aroramrinaal** [comment]
@bhavyastar @mowies , is this issue still open?

**Comment by StackScribe** [comment]
Keptn V1 has reached EOL on December 22nd, 2023, so this PR has been closed.

**Pull request closed without merging**
